### PR TITLE
fix: correct class on exception text of CommandCollection->add()

### PR DESCRIPTION
**Pull request by rkaiser0324**
The exception text in `add()` references the wrong class path; likely a typo.  See point 2 of discussion here: https://stackoverflow.com/a/69328477/1282424

### CODE REVIEW

File: src/Console/CommandCollection.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function add(string $name, $command)
             $class = is_string($command) ? $command : get_class($command);
             throw new InvalidArgumentException(sprintf(
                 "Cannot use '%s' for command '%s'. " .
-                "It is not a subclass of Cake\Console\Shell or Cake\Command\CommandInterface.",
+                "It is not a subclass of Cake\Console\Shell, nor does it implement Cake\Console\CommandInterface.",
```
Comment (ADmad) [inline src/Console/CommandCollection.php]: While this is technically correct, from a practical POV stating that it's not a subclass of `Cake\Command\Command` would make it more obvious for the user regarding how to remedy the situation.

Comment (dereuromark) [reply src/Console/CommandCollection.php]: `Cake\Command\Command` definitely sounds like the more helpful term to use here.

Comment (ADmad) [inline src/Console/CommandCollection.php]:
```suggestion
                "It is not a subclass of `Cake\Console\Shell` or `Cake\Command\Command`.",
```

Comment (rkaiser0324) [reply src/Console/CommandCollection.php]: @ADmad how about this instead:

`"It is not a subclass of Cake\Console\Shell or Cake\Command\Command."`

Comment (ADmad) [reply src/Console/CommandCollection.php]: That's pretty much what I already suggested in follow up comment :).

File: src/Console/CommandCollection.php [hunk 2]
```diff
@@ -67,7 +67,7 @@ public function add(string $name, $command)
             $class = is_string($command) ? $command : get_class($command);
             throw new InvalidArgumentException(sprintf(
                 "Cannot use '%s' for command '%s'. " .
-                "It is not a subclass of Cake\Console\Shell or Cake\Command\CommandInterface.",
+                "It is not a subclass of Cake\Console\Shell, or Cake\Command\Command.",
```
Comment (dereuromark) [inline src/Console/CommandCollection.php]: The comma is not needed IMO

**Comment by markstory** [comment]
Thank you 👍

**Pull request merged**